tests/integration/jobs/tasks.spec.ts

```diff
@@ -63,23 +64,25 @@ describe('tasks', function () {
 
     it('should create multiple tasks and return status code 201 and the created tasks ids', async function () {
       const taskId2 = '6f3669b8-8a65-4581-a127-3d26332635ed';
-      const createTaskModel1 = {
+      const createTaskModel1: ICreateTaskBody = {
         description: '1',
         parameters: {
           a: 2,
         },
         reason: '3',
-        percentage: 4,
         type: '5',
+        status: OperationStatus.IN_PROGRESS,
+        attempts: 0,
       };
-      const createTaskModel2 = {
+      const createTaskModel2: ICreateTaskBody = {
         description: '6',
         parameters: {
           b: 7,
         },
         reason: '8',
-        percentage: 9,
```

Comment:
why was this removed?
it is valid input, test should check mapping of as many fields as possible